cli/smartcontract/smart_contract.go

```diff
@@ -460,7 +464,7 @@ func testInvokeScript(ctx *cli.Context) error {
 	}
 
 	scriptHex := hex.EncodeToString(b)
-	resp, err := c.InvokeScript(scriptHex)
+	resp, err := c.InvokeScript(scriptHex, nil)
```

Comment:
Should we accept hashes as new cli parameter?

Reply:
That's a good idea, but I'm not sure what's the best way to add it, let me check that again.

Reply:
The way it is now:
```
USAGE:
   neo-go contract testinvokefunction -e endpoint scripthash [method] [arguments...]
```
We obviously can't make it
```
   neo-go contract testinvokefunction -e endpoint scripthash [method] [arguments...] [cosigners...]
```
but we have some options still, like:
1. Adding a flag (`blah-blah [-c cosigner] scripthash [method] [arguments...]`), but a flag can only have one value with our cli library, so it won't work for multiple cosigners.
2. Adding a separator flag (`blah-blah scripthash [method] [arguments...] [--] [cosigners...]`), makes `--` special, but should work fine.
3. Adding a specific argument type for cosigners (`blah-blah scripthash [method] [arguments and cosigners...]`), like `cosigner:value`, a bit more to type and a little uneasy to parse if one is to really mix arguments and cosigners, but should also work fine.

As for cosigner value syntax, we can support `hash[:scope]` in the CLI making a simple hash-only requests when only `hash` is provided and an extended scoped if scope is in the game too (but with a notice that it's only supported by neo-go nodes at the moment).

Reply:
Done the second version.